frontend/packages/ceph-storage-plugin/console-extensions.json

```diff
@@ -235,5 +235,15 @@
       "contextId": "csv-actions",
       "provider": { "$codeRef": "actions.useCsvActions" }
     }
+  },
+  {
+    "type": "console.dashboards/project/overview/item",
+    "properties": {
+      "model": { "$codeRef": "models.NooBaaObjectBucketClaimModel" },
```

Comment:
Don't we want to do something like this for model:
```
      "model": {
        "group": "objectbucket.io",
        "kind": "ObjectBucketClaim",
        "version": "v1alpha1"
      }
```
That will be consistent with most of the other extensions e.g `console.navigation/resource-ns`

Reply:
The inventory card can locate the model from its group/version/kind. It would be better to be consistent with how we handle it for nav. Also, accepting group/version/kind allows a plugin to add an item without needing to define its own model, instead using the model from discovery.